compiler/common-artifacts/CMakeLists.txt

```diff
@@ -1,5 +1,45 @@
-# TODO add pbtxt
+#[[ Generate common python virtual enviornment ]]
+find_package(PythonInterp 3 QUIET)
+find_package(PythonLibs 3 QUIET)
+
+if(NOT ${PYTHONINTERP_FOUND})
+  message(STATUS "Build common-artifacts: FALSE (Python3 is missing)")
+  return()
+endif()
+
+if(${PYTHON_VERSION_MINOR} LESS 3)
+  message(STATUS "Build common-artifacts: FALSE (You need to install Python version higher than 3.3)")
+  return()
+endif()
+
+# Create python virtual environment with tensorflow 1.13.1
+set(VIRTUALENV "${CMAKE_CURRENT_BINARY_DIR}/venv_1_13_1")
 
+add_custom_command(
+  OUTPUT ${VIRTUALENV}
+  COMMAND ${PYTHON_EXECUTABLE} -m venv ${VIRTUALENV}
+)
+
+# Create requirements.txt and install required pip packages
+set(REQUIREMENTS_FILE "requirements.txt")
```

Comment:
Is it ok with this PR without `requirements.txt` file?

Reply:
It's okay because I just create that file below by echo